app/models/miq_smart_proxy_worker.rb

```diff
@@ -1,9 +1,15 @@
 class MiqSmartProxyWorker < MiqQueueWorkerBase
+  include MiqWorker::ReplicaPerWorker
+
   require_nested :Runner
 
   self.required_roles       = ["smartproxy"]
   self.default_queue_name   = "smartproxy"
 
+  def self.supports_container?
+    true
+  end
```

Comment:
I'm not sure if this was an oversight or there was a reason for the smart proxy worker not running on pods

Reply:
Other than this worker, only the cockpit worker doesn't support containers:
```
>> MiqWorker.leaf_subclasses.reject(&:supports_container?).map(&:name)
=> ["MiqCockpitWsWorker"]
```

Maybe better to change the default in the base class to true and just say not supported in the cockpit worker?

Reply:
Good question.  Maybe we didn't enable the smart proxy because we didn't know if we could scan things in pods? 

https://github.com/ManageIQ/manageiq-pods/issues/595
https://github.com/ManageIQ/manageiq-pods/issues/596

Reply:
If we want to support it by default in all workers other than cockpit and possibly this one, yeah, I agree, we should default to support in the base class and have them disable it in those one or two classes.

Reply:
Yeah if that's the reason then that's only RHV fleecing the rest are fine (this happened to be Azure SSA)

Reply:
I don't see any discussion on https://github.com/ManageIQ/manageiq/pull/15884 about which works to enable and why